### PR TITLE
[minizip] init required by #1369

### PR DESCRIPTION
**Pull request by atkawa7**

### CODE REVIEW

**Comment by jasjuang** [comment]
Is there something wrong with their original CMakeLists? If there is, isn't it better to submit a PR for the original repo and once merged ask for a new release?

**Comment by atkawa7** [comment]
@jasjuang That is good question. It seems they didn't include minizip in the CMakeLists.txt. The reason I do not know. My assumption is since its a contributed project maybe they expect users to build it by themselves which I of coz I might be wrong.

File: ports/minizip/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.8)
+project(minizip C CXX)
```
Comment (codicodi) [inline ports/minizip/CMakeLists.txt]: No need to pick up C++ compiler, minizip is C-only

File: ports/minizip/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+set(MIN_SRC contrib/minizip)
+
+include_directories(${MIN_SRC} ${ZLIB_INCLUDE_DIRS} ${BZIP2_INCLUDE_DIR})
+
+set(SRC
+  ${MIN_SRC}/ioapi.c
+  ${MIN_SRC}/iowin32.c
+  ${MIN_SRC}/unzip.c
+  ${MIN_SRC}/zip.c
```
Comment (codicodi) [inline ports/minizip/CMakeLists.txt]: Any reason `mztools.c` is not included here?

File: ports/minizip/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+)
+set(HEADERS
+  ${MIN_SRC}/ioapi.h
+  ${MIN_SRC}/iowin32.h
+  ${MIN_SRC}/unzip.h
+  ${MIN_SRC}/zip.h
```
Comment (codicodi) [inline ports/minizip/CMakeLists.txt]: Same for `mztools.h`

File: ports/minizip/CMakeLists.txt (hunk 1 above)
Comment (atkawa7) [inline ports/minizip/CMakeLists.txt]: @codicodi Removing CXX causes linker errors

**Comment by ras0219-msft** [comment]
Looks good to me, thanks for the PR!

**Comment by atkawa7** [comment]
@jasjuang @ras0219-msft I just realized libkml is expecting a different minizip. Its expecting the fork of minizip https://github.com/nmoinvaz/minizip. Should I go ahead and PR for the fork?

**Pull request merged**